sampler/src/AgateEnvironment/AgateEnvironment.js

```diff
@@ -129,22 +143,34 @@ const StorybookDecorator = (story, config) => {
 		}
 	};
 	const skinFromURL = getPropFromURL('skin');
-	const currentSkin = skinFromURL ? skinFromURL : 'gallium';
+	const currentSkin = skinFromURL ? skinFromURL : Config.defaultProps.skin;
 	const newSkin = (memory.skin !== currentSkin);
 	memory.skin = currentSkin;  // Remember the skin for the next time we load.
 	const accentFromURL = getPropFromURL('accent');
 	const highlightFromURL = getPropFromURL('highlight');
+	const allSkins = boolean('show all skins', Config);
+	const skinKnobs = {};
+	if (!allSkins) {
+		skinKnobs.skin = select('skin', skins, Config, currentSkin);
```

Comment:
Why can't 'all skins' be a skin selection instead of a separate boolean toggle?

Reply:
This just seemed clearer. If you really want it that way, I can convert it over I guess.

Reply:
I'm not a big fan of disappearing knobs but I don't feel _super_ strongly about it.